ethcore/src/client/client.rs

```diff
@@ -439,7 +439,10 @@ impl<V> BlockChainClient for Client<V> where V: Verifier {
 		};
 		// that's just a copy of the state.
 		let mut state = self.state();
-		let sender = try!(t.sender());
+		let sender = try!(t.sender().map_err(|e| {
+			warn!(target: "evm", "Transaction mallformed: {:?}", e);
+			ExecutionError::TransactionMallformed
```

Comment:
And by implementing `From` trait you should be able to get rid of `map_err` functions (unless we want to print warning here - altough it could also be a side effect in `From::from` implementation (not sure if it's a good practice though))